pinot-core/src/main/java/org/apache/pinot/core/data/manager/BaseTableDataManager.java

```diff
@@ -347,19 +349,21 @@ public void reloadSegment(String segmentName, IndexLoadingConfig indexLoadingCon
         }
         indexDir = downloadSegment(segmentName, zkMetadata);
       } else {
-        LOGGER.info("Reload existing segment: {} of table: {}", segmentName, _tableNameWithType);
+        LOGGER.info("Reload existing segment: {} of table: {} currently on tier: {}", segmentName, _tableNameWithType,
```

Comment:
(minor) In most of the cases, tier will be `null`. Consider adding an if check instead of always logging the tier info